### PR TITLE
[Feature] About Team

### PR DESCRIPTION
**Pull request by gertie-sheshe**
## Description

Add The Team Section.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)

## Screenshots
Screenshot 2021-08-11 at 15 53 32
Screenshot 2021-08-11 at 15 53 47
Screenshot 2021-08-11 at 15 53 59

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: src/pages/index.js [hunk 1]
```diff
@@ -24,6 +25,7 @@ export default function Home({ boundary, blocks, ...props }) {
       <InsightData {...blocks?.dataInsights} />
       <DataVisuals {...blocks?.dataVisuals} />
       <ExploreOtherTools {...blocks?.exploreOtherTools} />
+      <AboutTeam {...aboutTeam} />
```
Comment (KhadijaMahanga) [inline src/pages/index.js:28]: About page

File: src/pages/index.js [hunk 2]
```diff
@@ -1,14 +1,15 @@
...
+import AboutTeam from "@/pesayetu/components/AboutTeam";
 import DataVisuals from "@/pesayetu/components/DataVisuals";
 import ExploreOtherTools from "@/pesayetu/components/ExploreOtherTools";
 import Hero from "@/pesayetu/components/Hero";
 import HowItWorks from "@/pesayetu/components/HowItWorks";
 import InsightData from "@/pesayetu/components/InsightsData";
 import Page from "@/pesayetu/components/Page";
 import PartnersAndNewsletter from "@/pesayetu/components/Partners";
-import { searchArgs } from "@/pesayetu/config";
+import { searchArgs, aboutTeam } from "@/pesayetu/config";
```
Comment (KhadijaMahanga) [inline src/pages/index.js:12]: cms integration

File: src/components/AboutTeamCard/index.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  title: PropTypes.string,
+};
+
+AboutTeamCard.defaultProps = {
+  image: undefined,
+  description: undefined,
+  title: undefined,
+};
+
+export default AboutTeamCard;
```
Comment (KhadijaMahanga) [inline src/components/AboutTeamCard/index.js:44]: Similar to insight cad component

File: src/pages/index.js (hunk 1 above)
Comment (gertie-sheshe) [reply src/pages/index.js]: Fixed

File: src/pages/index.js (hunk 2 above)
Comment (gertie-sheshe) [reply src/pages/index.js]: Fixed

File: src/components/AboutTeamCard/index.js (hunk 3 above)
Comment (gertie-sheshe) [reply src/components/AboutTeamCard/index.js]: Fixed

File: src/components/AboutTeam/index.js [hunk 4]
```diff
@@ -0,0 +1,83 @@
+import { Typography } from "@material-ui/core";
+import PropTypes from "prop-types";
+import React from "react";
+import Carousel from "react-multi-carousel";
+
+import useStyles from "./useStyles";
+
+// import AboutTeamCard from "@/pesayetu/components/AboutTeamCard";
```
Comment (kilemensi) [inline src/components/AboutTeam/index.js:8]:
```suggestion
```

File: src/components/AboutTeam/useStyles.js [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+    },
+    [breakpoints.up("lg")]: {
+      height: typography.pxToRem(278),
+      width: typography.pxToRem(278),
+    },
+  },
+  title: {
+    fontWeight: 900,
+    textAlign: "center",
+    textTransform: "uppercase",
```
Comment (kilemensi) [inline src/components/AboutTeam/useStyles.js:40]: Check in the theme, these should be set already.

File: src/components/AboutTeam/useStyles.js [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+    backgroundColor: "unset",
+    maxWidth: typography.pxToRem(350),
+    padding: `0 ${typography.pxToRem(20)}`,
+    [breakpoints.up("md")]: {
+      maxWidth: typography.pxToRem(296),
+      marginRight: typography.pxToRem(16),
+      padding: 0,
+    },
+    [breakpoints.up("lg")]: {
+      maxWidth: typography.pxToRem(376),
```
Comment (kilemensi) [inline src/components/AboutTeam/useStyles.js:22]: Where do these maxWidths come from and why are we using them? For example, in mobile, don't we want the card to be based on screen size?

File: src/components/AboutTeam/index.stories.js [hunk 7]
```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+import AboutTeam from ".";
+
+import { aboutTeam } from "@/pesayetu/config";
+
+export default {
+  title: "Sections/About Team",
```
Comment (kilemensi) [inline src/components/AboutTeam/index.stories.js:8]:
```suggestion
  title: "Sections/AboutTeam",
```

**Review by kilemensi — COMMENTED** [review]
👍🏾

File: src/components/AboutTeam/index.stories.js (hunk 7 above)
Comment (gertie-sheshe) [reply src/components/AboutTeam/index.stories.js]: Fixed

File: src/components/AboutTeam/useStyles.js (hunk 6 above)
Comment (gertie-sheshe) [reply src/components/AboutTeam/useStyles.js]: Fixed

File: src/components/AboutTeam/useStyles.js (hunk 5 above)
Comment (gertie-sheshe) [reply src/components/AboutTeam/useStyles.js]: Fixed

File: src/components/AboutTeam/useStyles.js [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+    boxShadow: "none",
+    backgroundColor: "unset",
+    padding: `0 ${typography.pxToRem(20)}`,
+    [breakpoints.up("md")]: {
+      marginRight: typography.pxToRem(16),
+      padding: 0,
+    },
+  },
+  cardMedia: {
+    height: typography.pxToRem(278),
```
Comment (kilemensi) [inline src/components/AboutTeam/useStyles.js:21]: Looking at the XD, the image is limited to width of 278 even in mobile

File: src/components/AboutTeam/useStyles.js [hunk 9]
```diff
@@ -0,0 +1,70 @@
...
+      paddingBottom: typography.pxToRem(80),
+    },
+  },
+  container: {
+    flexDirection: "column",
+    [breakpoints.up("md")]: {
+      flexDirection: "row",
+      justifyContent: "space-between",
+    },
+  },
```
Comment (kilemensi) [inline src/components/AboutTeam/useStyles.js:51]:
```suggestion
```

**Review by kilemensi — COMMENTED** [review]
👍🏾

File: src/components/AboutTeam/useStyles.js (hunk 8 above)
Comment (gertie-sheshe) [reply src/components/AboutTeam/useStyles.js:21]: Fixed

File: src/components/AboutTeam/useStyles.js [hunk 10]
```diff
@@ -0,0 +1,70 @@
...
+    height: typography.pxToRem(278),
+    position: "relative",
+    [breakpoints.up("md")]: {
+      height: typography.pxToRem(278),
+      width: typography.pxToRem(278),
+    },
+    [breakpoints.up("lg")]: {
+      height: typography.pxToRem(278),
+      width: typography.pxToRem(278),
+    },
```
[the remark this answers is not in the document]
Comment (gertie-sheshe) [reply src/components/AboutTeam/useStyles.js]: Fixed

**Review by BrendaBucha — APPROVED** [review]
@gertie-sheshe Looks good

**Review by KhadijaMahanga — APPROVED** [review]
🚀

**Review by KobbyMmo — APPROVED** [review]
looks good @gertie-sheshe

File: src/components/AboutTeam/useStyles.js [hunk 11]
```diff
@@ -0,0 +1,50 @@
...
+    backgroundColor: palette.background.paper,
+    padding: `${typography.pxToRem(40)} 0`,
+    [breakpoints.up("md")]: {
+      padding: `${typography.pxToRem(80)} 0`,
+    },
+  },
+  cardRoot: {
+    boxShadow: "none",
+    backgroundColor: "unset",
+    padding: `0 ${typography.pxToRem(20)}`,
```
Comment (kilemensi) [inline src/components/AboutTeam/useStyles.js:14]: Where is this padding from?

**Review by kilemensi — COMMENTED** [review]
👍🏾 

-----

Centre on mobile
Screenshot 2021-08-20 at 10-56-00 PesaYetu

**Pull request merged**